cmd/captplanet/run.go

```diff
@@ -131,7 +131,7 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 			o,
 			// TODO(coyle): re-enable the checker and repairer after we determine why it is panicing
 			// runCfg.Satellite.Checker,
-			// runCfg.Satellite.Repairer,
+			runCfg.Satellite.Repairer,
```

Comment:
Should we update the TODO comment above?

Reply:
Oh. I simply meant we can update the comment to reflect that the repairer is now re-enabled, i.e. the comment should mentioned just the checker. The repairer is not panicking any more, right?

Reply:
done